pipelines/pingcap/tiflash/release-8.1/pod-pull_unit-test.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+    - name: runner
+      image: "hub.pingcap.net/tiflash/tiflash-llvm-base:amd64-llvmorg-14.0.6"
+      command:
+        - "cat"
+      tty: true
+      resources:
+        requests:
+          memory: "32Gi"
+          cpu: "12000m"
+      volumeMounts:
+      - mountPath: "/home/jenkins/agent/rust"
+        name: "volume-0"
+        readOnly: false
+      - mountPath: "/home/jenkins/agent/ccache"
+        name: "volume-1"
+        readOnly: false
+      - mountPath: "/home/jenkins/agent/dependency"
+        name: "volume-2"
+        readOnly: false
+      - mountPath: "/home/jenkins/agent/ci-cached-code-daily"
+        name: "volume-4"
+        readOnly: false
+      - mountPath: "/home/jenkins/agent/proxy-cache"
+        name: "volume-5"
+        readOnly: false
+      - mountPath: "/tmp"
+        name: "volume-6"
+        readOnly: false
+      - mountPath: "/tmp-memfs"
+        name: "volume-7"
+        readOnly: false
+      - mountPath: "/home/jenkins"
+        name: "volume-8"
+        readOnly: false
+    - name: jnlp  # TODO: remove this default container
+      image: "jenkins/inbound-agent:3148.v532a_7e715ee3-1"
+      resources:
+        requests:
+          cpu: "500m"
+          memory: "500Mi"
+        limits:
+          cpu: "500m"
+          memory: "500Mi"
+      volumeMounts:
+      - mountPath: "/home/jenkins/agent/rust"
+        name: "volume-0"
+        readOnly: false
+      - mountPath: "/home/jenkins/agent/ccache"
```

Comment:
@purelind we should remove this container. It just a commander container which Jenkins will append it automatically.

Reply:
There is a cache presence check in the pipeline that will definitely be executed in the jnlp container, so it cannot be removed for now.